core-web/libs/ui/src/lib/components/dot-copy-button/dot-copy-button.component.ts

```diff
@@ -31,20 +21,42 @@ import { DotClipboardUtil } from '../../services/clipboard/ClipboardUtil';
     imports: [TooltipModule, ButtonModule, NgIf],
     changeDetection: ChangeDetectionStrategy.OnPush
 })
-export class DotCopyButtonComponent implements OnInit {
-    @Input() copy = '';
-    @Input() label: string;
-    @Input() tooltipText: string;
+export class DotCopyButtonComponent {
+    /**
+     * String to be copied to clipboard
+     */
+    copy = input('');
```

Comment:
It seems to me that we agreed not to use `signal inputs` until they leave developer preview

This component is not that big and I think there will be no problem, but be careful with that because the API could change